src/components/Error/Error.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import './Error.css';
 import Header from '../Header/Header';
 
-const Error = () => {
-  return (
-    <>
-      <Header />
-      <div className='error-messaging'>
-        <h4>Thank you for visiting our website.</h4>
-        <p>You've found a page that we cannot find.</p>
-        <p>Please return to our 🏡 home page using the Treezy link above.</p>
-      </div>
-    </>
-  )
-}
+const Error = ({ clearZipTrees }) => {
+	return (
+		<>
+			<Header clearZipTrees={clearZipTrees} />
+			<div className="error-messaging">
+				<h4>Thank you for visiting our website.</h4>
+				<p>You've found a page that we cannot find.</p>
+				<p>Please return to our 🏡 home page using the Treezy link above.</p>
+			</div>
+		</>
+	);
```

Comment:
There seems to be some indentation differences in the Error component.  Did we want the indentation like this?